src/servsock.js

```diff
@@ -0,0 +1,53 @@
+var fs = require('fs');
+var net = require('net');
+
+export default class ServiceSocket {
```

Comment:
Does this class need error event handlers?  In case the client connection breaks or something.